python/cudf/cudf/io/parquet.py

```diff
@@ -669,6 +690,7 @@ def to_parquet(
     index=None,
     partition_cols=None,
     partition_file_name=None,
+    partition_offsets=None,
     statistics="ROWGROUP",
     metadata_file_path=None,
```

Comment:
We will also want the docstring to explain that `metadata_file_path` should be a list of the same length of `partition_offsets` (when partition_offsets is specified).